polynote-runtime/src/main/scala/polynote/runtime/macros/ExpandedScopeMacros.scala

```diff
@@ -55,6 +55,8 @@ class ExpandedScopeMacros(val c: whitebox.Context) {
       }
     }
 
+    // TODO: something is weird with this cache... the second time around it doesn't find the right reprs!
+    //       Fix it! (context: It didn't seem to be a problem before the plotting stuff)
```

Comment:
Not sure about whether these reprs might be requested concurrently but if so, maybe it needs to be changed to a ConcurrentHashMap?

Reply:
Hmm... I suppose it's possible, though I think there should probably only be one thread compiling Scala code at any given time with the same `Global`.

Anyway I should remove that comment, since it didn't turn out to be the reason that it stopped working the second time.